plugins/org.python.pydev.ast/src/org/python/pydev/ast/interpreter_managers/InterpreterInfo.java

```diff
@@ -1631,7 +1631,7 @@ public void setEnvVariables(String[] env) {
             File condaPrefix = this.getCondaPrefix();
             if (condaPrefix == null) {
                 Log.log("Unable to find conda prefix for: " + this.getExecutableOrJar());
-            } else if (new File(condaPrefix, "/etc/conda/activate.d").exists()) {
+            } else if (new File(condaPrefix, "/").exists()) {
                 try {
                     String[] cmdLine;
                     File relativePath;
```

Comment:
Another thing: we should create a new Map (`private Map<String,String> condaEnv = new HashMap<>()`) and use it instead of reactivating the environment every time (i.e.: the information should be cached instead of reactivating the environment every time).

It should be reset when `setModificationStamp` is called (use `condaEnv = new HashMap<>()` instead of `condaEnv.clear()` to avoid racing conditions). Also, always create a new map and assign to the variable instead of mutating the existing one and mark it as `volatile`.